commcare_export/cli.py

```diff
@@ -73,6 +73,8 @@ def add_to_parser(self, parser):
                  help="When saving to a SQL database don't allow changing column types once they are created."),
         Argument('missing-value', default=None, help="Value to use when a field is missing from the form / case."),
         Argument('batch-size', default=1000, help="Number of records to process per batch."),
+        Argument('checkpoint-key', help="Use this key for all checkpoints instead of the query file MD5 hash "
+                                        "in order to prevent table rebuilds after a query file has been edited."),
```

Comment:
Yea I'm sure there are but I'd rather leave it to the user to determine since the rules could get quite complicated and may become opaque to the user.